src/app.ts

```diff
@@ -107,8 +107,14 @@ async function app() {
 
   try {
     const rawFSH = getRawFSHes(input);
-    if (rawFSH.length === 0) {
-      logger.info('No FSH files present.');
+    if (
+      rawFSH.length === 0 &&
+      !fs.existsSync(path.join(originalInput, 'config.yaml')) &&
+      !fs.existsSync(path.join(originalInput, 'sushi-config.yaml')) &&
+      !fs.existsSync(path.join(originalInput, 'fsh', 'config.yaml')) &&
+      !fs.existsSync(path.join(originalInput, 'fsh', 'sushi-config.yaml'))
+    ) {
+      logger.info('No FSH files or sushi-config.yaml present.');
       process.exit(0);
     }
     config = readConfig(isIgPubContext ? originalInput : input, isLegacyIgPubContext);
```

Comment:
Would it be simpler to move the check you've added above below line 114, where the config is read in? That way couldn't you just check if the config existed, and when we deprecate `config.yaml`, we don't have it being checked for in two places.

That would mean that SUSHI would still run if it extracts the configuration from an IG resource, or if it creates one from a package.json, so if we don't want that, then we can just keep it as is.

Reply:
I had considered that, but the problem is that if no config is found, `readConfig` logs an error and throws.  The behavior we want is that if there are no FSH files and no config, we exit gracefully (because it's not a FSH project) -- so logging an error and throwing would make for an ungraceful exit.